unit-tests/native/src/test/scala-3/scala/scala/scalanative/runtime/Continuations.scala

```diff
@@ -0,0 +1,96 @@
+package scala.scalanative.runtime
+
+import org.junit.Test
+import org.junit.Assert._
+
+import Continuations._
+import scala.scalanative.meta.LinktimeInfo.{isWindows, is32BitPlatform}
+
+class Scala3ContinuationsTests:
+  @Test def canBoundaryNoSuspend() =
+    if !isWindows then
+      val res = boundary[Int] {
+        val x = 1
+        val y = 2
+        x + y
+      }
+      assert(res == 3)
+  @Test def canBoundarySuspend() =
+    if !isWindows then
+      val res = boundary[Int] {
+        val x = 1
+        suspend[Int](_ => x + 1)
+        ???
+      }
+      assert(res == 2)
+  @Test def canBoundarySuspendImmediateResume() =
+    if !isWindows then
+      val r = boundary[Int] {
+        1 + suspend[Int, Int](r => r(2)) + suspend[Int, Int](r => r(3)) + 4
+      }
+      assert(r == 10)
+  @Test def canBoundarySuspendCommunicate() =
```

Comment:
The tests above are short so they work good with breacless syntax, but the tests below are a bit long, have multiple nested scopes, so it would be cleaner to apply the braces here. Also it would be great to put blank lines before each test case above (between end of test body and new test definition)